### DISCUSSION TITLE
How to create a corpus with multiple docs?

### DISCUSSION THREAD

**Discussion by fatihbozdag** [explosion/spaCy#6751] [Help: Coding & Implementations]
Greetings;

I have 25 text files from different sources and I would like to create a corpus with 25 docs each representing one of the text files in the list so that I can cross-compare results in different docs.

For instance;

```
list = [text1.txt, text2.txt, text3.txt....]

Corpus = nlp(text_data of text files in the list)

Corpus = [Doc1, Doc2, Doc3....]

```
I would like to iterate the token over docs so that I would observe how a token/lemma is used over docs. 

Is it possible?

**Comment by adrianeboyd** [comment]
There are a lot of different options here, but one I can suggest is `textacy`, which has a `Corpus` class that may be the kind of thing you're looking for: https://textacy.readthedocs.io/en/stable/quickstart.html#make-a-corpus

**Reply by fatihbozdag** [reply]
Yeah, textacy is great but all I have raw text files. Textacy is able to create multiple docs if texts are stated within csv files. Is there a way to import text files in that way?
